personalcapital.js

```diff
@@ -94,6 +100,16 @@ function getBlockcypherBalance(url) {
     });
 }
 
+function getBlockcypherBalance(url) {
+    return getJSON(url).then(function(data) {
+        if (data.balance) {
+            return data.balance;
+        } else {
+            throw "Invalid json response";
```

Comment:
Does "throw" trigger the reject case on L120 here?: https://github.com/sclem/pfcrypto/pull/5/files#diff-7d9c8f7f3ca2f4e8efdbeba4a3169957R120